src/modules/75-cf/components/FlagActivation/FlagPrerequisites.tsx

```diff
@@ -397,58 +369,55 @@ export const FlagPrerequisites: React.FC<FlagPrerequisitesProps> = props => {
     : undefined
 
   return (
-    <Container className={cx(css.collapseFeatures, css.module)}>
-      <Collapse {...editCardCollapsedProps} heading={prerequisitesTitle}>
-        {!!featureFlag.prerequisites?.length && (
-          <Layout.Horizontal flex margin={{ bottom: 'xsmall' }}>
-            <Text width="50%">{getString('flag')}</Text>
-            <Text width="50%">{getString('cf.shared.variation')}</Text>
+    <Layout.Vertical margin={{ bottom: 'medium', top: 'medium' }}>
+      <Layout.Vertical margin={{ bottom: 'medium' }}>
+        <Heading level={5} font={{ variation: FontVariation.H5 }}>
+          <StringWithTooltip stringId="cf.shared.prerequisites" tooltipId="ff_ffPrerequisites_heading" />
+        </Heading>
+        <Text font={{ variation: FontVariation.SMALL_SEMI }} color={Color.GREY_500}>
+          {getString('cf.featureFlags.prerequisitesDesc')}
+        </Text>
+      </Layout.Vertical>
+      {featureFlag.prerequisites?.length && (
```

Comment:
Little gotcha here. If `featureFlag.prerequisites.length` is `0`, it'll actually output that on screen. Use a `!!` to make sure it's a boolean at this stage
```suggestion
      {!!featureFlag.prerequisites?.length && (
```